lib/components/table.vue

```diff
@@ -4,91 +4,98 @@
-                    @keydown.space.stop.prevent="headClicked($event,field,key)"
-                    :key="key"
-                    :class="fieldClass(field,key)"
-                    :style="field.thStyle || {}"
-                    :aria-label="field.sortable ? ((sortDesc && sortBy === key) ? labelSortAsc : labelSortDesc) : null"
-                    :aria-sort="(field.sortable && sortBy === key) ? (sortDesc ? 'descending' : 'ascending') : null"
-                    :tabindex="field.sortable?'0':null"
-                >
-                  <slot v-if="$scopedSlots['FOOT_'+key]" :name="'FOOT_'+key" :label="field.label" :column="key" :field="field">
+        <tr>
+            <th v-for="(field,key) in fields"
+                @click.stop.prevent="headClicked($event,field,key)"
+                @keydown.enter.stop.prevent="headClicked($event,field,key)"
+                @keydown.space.stop.prevent="headClicked($event,field,key)"
+                :key="key"
+                :class="fieldClass(field,key)"
+                :style="field.thStyle || {}"
+                :aria-label="field.sortable ? ((sortDesc && sortBy === key) ? labelSortAsc : labelSortDesc) : null"
+                :aria-sort="(field.sortable && sortBy === key) ? (sortDesc ? 'descending' : 'ascending') : null"
+                :tabindex="field.sortable?'0':null"
+            >
+                <slot v-if="$scopedSlots['FOOT_'+key]" :name="'FOOT_'+key" :label="field.label" :column="key"
+                      :field="field">
                     <div v-html="field.label"></div>
-                  </slot>
-                  <slot v-else :name="'HEAD_'+key" :label="field.label" :column="key" :field="field">
+                </slot>
+                <slot v-else :name="'HEAD_'+key" :label="field.label" :column="key" :field="field">
                     <div v-html="field.label"></div>
-                  </slot>
-                </th>
-            </tr>
+                </slot>
+            </th>
+        </tr>
         </tfoot>
         <tbody>
-            <tr v-for="(item,index) in _items"
-                :key="index"
-                :class="rowClass(item)"
-                @click="rowClicked($event,item,index)"
-                @hover="rowHovered($event,item,index)"
-            >
-                <td v-for="(field,key) in fields" :key="key" :class="tdClass(field, item, key)">
-                    <slot :name="key" :value="item[key]" :item="item" :index="index">{{item[key]}}</slot>
+        <tr v-for="(item,index) in _items"
+            :key="index"
+            :class="rowClass(item)"
+            @click="rowClicked($event,item,index)"
+            @hover="rowHovered($event,item,index)"
+        >
+            <template v-for="(field,key) in fields">
+                <td v-if="hasFormatter(field)" :key="key" :class="tdClass(field, item, key)"
+                    v-html="callFormatter(field, item, key)">
```

Comment:
Although it might be handy to have it still in there for some people, but as the 3rd argument.